STYLEGUIDE.md

```diff
@@ -72,3 +78,74 @@ import 'camera.dart';
 import 'game.dart';
 import 'viewport.dart';
 ```
+
+### Identifiers
+
+Use all [effective dart patterns](https://dart.dev/guides/language/effective-dart)
+
+* Avoid using global constants, but if you do use, name it prefixing with a `k`.
+
+```dart
+// Do not:
+const someConstant = 42;
+
+// Do instead:
+const kSomeConstant = 42;
+```
+
+### Comments
+
+* Add an identifier for TODO comments
+* Use full capitalized TODO
+
+```dart
+
+// Do not:
+// Todo: This thing should be that thing
+final thisThing = 13;
+
+// Do instead:
+// TODO(wofstein): This thing should be that thing
+const thisThing = 13;
+```
+
+### Asserts
+
+* Use asserts to detect contract violation.
+* Use asserts to run debug only code
+
+Example:
+````dart
+void something(int smaller, int bigger) {
+  assert(small < bigger, "smaller is not smaller than bigger");
+  assert((){
+    // debug only code
+  });
+  
+  //...
+}
+
+````
+
+
+
+## Code documentation 
+
+Specific guides for dartdoc documentation.
+
+#### Use impersonal tone
+
+Avoid using words such as "you", "ours" and "yours" on code documentation
```

Comment:
I think we need to review the docs if we want to enforce this rule. Might be quick though.

Reply:
151 occurrences just for `you` and `yours`. I think I will do this in a  separate PR.

Reply:
I would omit the rule for now; in a followup we can add the rule + fix and/or discuss if we need to change everything

Reply:
Moved to an issue so we can discuss further and separately: https://github.com/flame-engine/flame/issues/767